airflow/utils/db.py

```diff
@@ -646,8 +646,8 @@ def create_default_connections(session: Session = NEW_SESSION):
     )
     merge_conn(
         Connection(
-            conn_id="tabular_default",
-            conn_type="tabular",
+            conn_id="iceberg_default",
+            conn_type="iceberg",
             host="https://api.tabulardata.io/ws/v1",
```

Comment:
This is the reason for the test failures.
This file is sorted by conn_id changing tabular to icebreg break the sorting.
Note also that the host needs to be changed.

I think it's best to leave tabular and just add iceberg along side.
We can remove tabular conn_id if/when we remove the tabular provider